WordPress/Classes/ViewRelated/Site Creation/Web Address/AddressTableViewCell.swift

```diff
@@ -0,0 +1,279 @@
+import UIKit
+import WordPressKit
+
+final class AddressTableViewCell: UITableViewCell {
+
+    // MARK: - Dependencies
+
+    private let domainPurchasingEnabled = FeatureFlag.siteCreationDomainPurchasing.enabled
+
+    // MARK: - Views
+
+    private var borders = [UIView]()
```

Comment:
This property is from the old implementation, and it should be removed once `Domain Purchasing Feature Flag` is `true`.